app/src/test/java/org/pesmypetcare/mypetcare/controllers/exercise/TestTrGetAllExercises.java

```diff
@@ -0,0 +1,79 @@
+package org.pesmypetcare.mypetcare.controllers.exercise;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.pesmypetcare.httptools.utilities.DateTime;
+import org.pesmypetcare.mypetcare.controllers.exercise.TrGetAllExercises;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redundant import from the same package - org.pesmypetcare.mypetcare.controllers.exercise.TrGetAllExercises.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5561369)